Create soul_soil.json

Katorone opened the pull request:
Copy of the centrifuge soul_sand recipe, but for soul_soil
I'm sorry, I don't know if I need to add a reference to this recipe anywhere else.

Commit:
Create soul_soil.json

Copy of the centrifuge soul_sand recipe, but for soul_soil
I'm sorry, I don't know if I need to add a reference to this recipe anywhere else.

Prospector commented:
If the recipe is identical, then it might be a better idea to make a tag for soul sand and soul soil, and use the tag in the recipe.

Commit:
Update soul_soil.json

Adjusted the recipe to return 8 minecraft:dirt instead of 10 minecraft:sand.  Lowered the amount of dirt, because it's more dynamic than sand for processing it further.  According to REI, this would be the first TR recipe to provide dirt. (besides the scrapbox)

ghost commented:
I've adjusted the recipe to return 8 minecraft:dirt instead of 10 minecraft:sand.

jusvit commented:
Alright when the dirt changes which makes more sense, everything else looks good.
